superset-frontend/src/SqlLab/components/SqlEditorLeftBar/SqlEditorLeftBar.test.jsx

```diff
@@ -58,7 +52,16 @@ fetchMock.get('glob:*/superset/tables/**', {
   tableLength: 1,
 });
 
+const setup = (props, store) =>
+  render(<SqlEditorLeftBar {...props} />, {
+    useRedux: true,
+    ...(store && { store }),
+  });
+
 describe('Left Panel Expansion', () => {
+  const renderAndWait = (props, store) =>
```

Comment:
There are some unnecessary uses of the await keyword in the tests. I created a patch file with some changes. To use it you need to uncompress the file and `git apply tests.patch`

[tests.patch.zip](https://github.com/apache/superset/files/9737632/tests.patch.zip)